src/components/AgenciesPage/services/fetchAgencies.ts

```diff
@@ -0,0 +1,11 @@
+import { Endpoints } from '../../../shared/api/constants/endpoints';
+import { axiosInstance } from '../../../shared/api/services/axiosInstance';
+
+export const fetchAgencies = async () => {
```

Comment:
there can be added as return type Promise<void>

Reply:
Agree
Done

Reply:
added

Reply:
added

Reply:
done